docs/client/initializing.md

```diff
@@ -0,0 +1,73 @@
+# <img align="center" src="../images/logo.png">  Initializing Your C# Client
+
+The first step to using your generated client in code is to import and initialize your client. Our SDKs are modelled such
+that the client is the main point of access to the generated code.
+
+## Initializing and Authenticating Your Client
+
+You can use your client if you import the namespace you specified when generating (under flag `--namespace`). For the sake of this example,
+let's say the namespace is `Azure.Pets` and your client is `PetsClient`. You could then access the client if you import the namespace.
+The default client also comes with these three parameters:
```

Comment:
We don't, currently generate a public ctor that configures the pipeline. The one that takes clientDiagnostics/pipeline is protected so the idea is that you have to define a constructor based on what authentication is to be used.

Reply:
For example https://github.com/Azure/azure-sdk-for-net/blob/3d8e6a42bd78716e9001ec54c5073429f8c9efe3/sdk/synapse/Azure.Analytics.Synapse.Monitoring/src/Customization/MonitoringClient.cs#L10

Reply:
We do it for ARM because we know that token credential-based auth is supported there